metadata/src/test/java/org/apache/kafka/metadata/PartitionRegistrationTest.java

```diff
@@ -371,7 +371,7 @@ public void testPartitionRegistrationToRecord_ElrShouldBeNullIfEmpty() {
             setPartitionEpoch(0);
         List<UnwritableMetadataException> exceptions = new ArrayList<>();
         ImageWriterOptions options = new ImageWriterOptions.Builder().
-            setMetadataVersion(MetadataVersion.IBP_3_8_IV0).
+            setMetadataVersion(MetadataVersion.IBP_3_8_IV1).
```

Comment:
Hmm, this is testing ELR. So it seems that it needs to be IBP_4_0_IV0. @CalvinConfluent : Could you confirm this?

Reply:
Yes, it requires IBP_4_0_IV0